cmd/acraserver/config.go

```diff
@@ -70,8 +71,26 @@ func NewConfig() *Config {
 
 var ErrTwoDBSetup = errors.New("only one db supported at one time")
 
-func (config *Config) SetFirewall(fw firewall.FirewallInterface) {
-	config.firewall = fw
+func (config *Config) SetFirewall(censorConfigPath string) error {
+	firewall := &firewall.Firewall{}
```

Comment:
better to assign one time `config.firewall = &firewall.Firewall{}` instead 5 extra lines